medusa/indexers/indexer_base.py

```diff
@@ -214,6 +214,26 @@ def _save_images(self, series_id, images):
                 res: resolution such as `1024x768`, `original`, etc
                 id: the image id
         """
+        def penalize_images(images):
+            """Penalize images that have a rating count of less then 5 and images that are not withing the highest
+            category of resolution, if there are multiple."""
+            images = list(images)  # We don't need to the generator.
```

Comment:
remove this line

Reply:
can't remove that line. As then after getting the set of bannertype2 the decorator has been used.
`banner_types = set([image['bannertype2'] for image in images])`

And I need to go through it twice.